config/locales/uk.yml

```diff
@@ -0,0 +1,49 @@
+uk:
+  errors:
+    messages:
+      taken_in_past: 'раніше використовувався.'
+      equal_to_current_password: 'має відрізнятися від поточного паролю.'
+      password_complexity:
+        digit:
+          one: 'повинен включати хоча б одну цифру'
+          few: 'повинен включати хоча б %{count} цифри'
+          many: 'повинен включати хоча б %{count} цифр'
+          other: 'повинен включати хоча б %{count} цифри'
+        lower:
+          one: 'повинен включати хоча б одну малу букву'
+          few: 'повинен включати хоча б %{count} малі букви'
+          many: 'повинен включати хоча б %{count} малих букв'
+          other: 'повинен включати хоча б %{count} малі букви'
+        symbol:
+          one: 'повинен включати хоча б один розділовий знак або символ'
+          few: 'повинен включати хоча б %{count} розділові знаки або символи'
+          many: 'повинен включати хоча б %{count} розділових знаків або символів'
+          other: 'повинен включати хоча б %{count} розділові знаки або символи'
+        upper:
+          one: 'повинен включати хоча б одну прописну букву'
+          few: 'повинен включати хоча б %{count} прописні букви'
+          many: 'повинен включати хоча б %{count} прописних букв'
+          other: 'повинен включати хоча б %{count} прописні букви'
+  devise:
+    invalid_captcha: 'Неправильне введення капчі.'
+    invalid_security_question: 'Неправильна відповідь на секретне питання.'
+    paranoid_verify:
+      code_required: 'Пожалуйста введите код, полученный от нашей команды поддержки'
+      code_required: 'Введіть, будь ласка, код від нашої команди підтримки'
+    paranoid_verification_code:
+      show:
+        submit_verification_code: Відправити код підтвердження
+        verification_code: Код підтвердження
+        submit: Відправити
+    password_expired:
+      updated: 'Ваш новий пароль збережено.'
+      change_required: 'Ваш пароль застарілий. Будь ласка, оновіть пароль.'
+      show:
+        renew_your_password: Оновити свій пароль
+        current_password: Теперішній пароль
+        new_password: Новий пароль
+        new_password_confirmation: Підтвердити новий пароль
+        change_my_password: Змінити пароль
+    failure:
+      session_limited: 'Ваші облікові дані були використані в іншому браузері. Будь ласка, авторизуйтесь знову щоб продовжити в цьому браузері.'
+      expired: "Ваш аккаунт застарів через неактивніть. Будь ласка, зв'яжіться з адміністратором."
```

Comment:
Is this a good suggestion from Google?

![image](https://user-images.githubusercontent.com/691365/95321379-fc73ca80-0868-11eb-9ea2-302e05a8b79c.png)

Reply:
typo, fixed